### ISSUE TITLE
[A11y] Toadt notification is  control type is set as tree

### ISSUE THREAD

**Issue by udeeshagautam** [microsoft/vscode#82728]
Environment Details:
Application Name: Visual Studio Code
Application Version: 1.38.1 (user setup)
Commit: b37e54
Electron: 4.2.10
Node.js: 10.11.0
OS: Windows_NT x64 10.0.18362

Steps to Reproduce:

1. Open any workspace with Extensions recommendations
2. Toast notification pops up
3. Open Accessibility insights and track the control type for toast notification.
4. Control type is tree when it is expected to be a dialog.

Accessibility Impact: the control is called out as tree by screen reader which might be confusing.

**Comment by bpasero** [comment]
It is true that each notification toast is a tree itself because that is the underlying widget we are using. We can possible change that if there is API from the tree level.

**Comment by isidorn** [comment]
There is already a [`getRole`](https://github.com/Microsoft/vscode/blob/99ba33dc9d17e09a5cf427dbb164777780ca8d11/src/vs/base/browser/ui/list/listView.ts#L47) in the AriaProvider which should be exposed in the `Tree`.
Not sure if you can use that one, or you need to introduce a new one `getSetRole` or the current one will be enough?
@bpasero can you try it out?

**Comment by bpasero** [comment]
@isidorn this was not possible without tree changes (and also a bugfix where the tree would always set the aria role "tree", even if options were provided). Please review https://github.com/microsoft/vscode/pull/82758

**Comment by isidorn** [comment]
Please note that this should now be fixed properly via 86db1ec2ac
So the parent is a `list` and each toast is a `dialog`
